app/controllers/coronavirus_form/expert_advice_type_controller.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class CoronavirusForm::ExpertAdviceTypeController < ApplicationController
+  include ActionView::Helpers::SanitizeHelper
+  include FieldValidationHelper
+
+  def show
+    session[:expert_advice] ||= []
```

Comment:
We're re-using the session key `expert_advice` so this should be e.g. `expert_advice_type`.